app/views/shared/error/not_found.html.erb

```diff
@@ -0,0 +1,6 @@
+<div>
+  <h1>Resource Not Found.</h1>
```

Comment:
Could you add these as translated strings

Reply:
Added in 77974b5a3aba27f4678193204483c14ee997f770